### PR TITLE
*: update pd api

### PR DESCRIPTION
**Pull request by rleungx**
# What problem does this PR solve? 
Our API is out of date.

### What is changed and how it works?
This PR is used to update PD API. It should be reviewed after #1602 is merged.

### Check List 

Tests 

 - No code

### CODE REVIEW

File: server/api/api.raml [hunk 1]
```diff
@@ -663,6 +667,41 @@ types:
       500:
         description: PD server failed to proceed the request.
 
+  /limit:
+    description: The all stores' balance rate limit.
```
Comment (disksing) [inline server/api/api.raml]: `The stores` or `All stores`.

File: server/api/api.raml [hunk 2]
```diff
@@ -663,6 +667,41 @@ types:
...
+          description: PD server failed to proceed the request.
+    post:
+      description: Set all stores' balance rate limit.
+      body:
+        application/json:
+          description: key-value pair.
+          type: object
+      responses:
+        200:
+          description: The all stores' balance rate limit is updated.
```
Comment (disksing) [inline server/api/api.raml]: `The stores` or `All stores`

File: server/api/api.raml (hunk 1 above)
Comment (dcalvin) [reply server/api/api.raml]: Suggest using: balance rate limit for all stores.

File: server/api/api.raml [hunk 3]
```diff
@@ -746,6 +785,22 @@ types:
         500:
           description: PD server failed to proceed the request.
 
+  /limit:
+    description: The specific store's balance rate limit.
```
Comment (dcalvin) [inline server/api/api.raml]:
```suggestion
    description: The balance rate limit for the specific store.
```

Comment (dcalvin) [reply server/api/api.raml]: 类似的都改下吧

**Pull request merged**